<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentRegistration</class>
 <widget class="QDialog" name="StudentRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3a3f5c;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LBTC SMART RECORD SYSTEM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_RegisterStudent_back">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
  </widget>
  <widget class="QPushButton" name="StudentRegistration_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>971</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">     
QPushButton{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 6px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>STUDENT REGISTRATION FORM</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>971</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Fname">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mname">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Lname">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Student_Program">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Program</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Student_ID">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Student ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Enro_year">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Year Enrolled</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Course_Duration">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Course Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Student_Status">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Student_browse_image">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>218</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Browse image</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </widget>
   <widget class="QLabel" name="image_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
 QLabel {
  border:2px solid green;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
 
};</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Student_Email">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Email Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Student_Phone">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 6px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>All Fields marked with * are required fields make sure to have them all filled</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_submit_lecturer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Reset_Lecturer">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Stud_NXTOF_KIN">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Next oF Kin Contact Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>737</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Year Enrolled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Student ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Program</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>160</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>230</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Email Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>160</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next OF KIN Contact Number#</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>160</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Sponsorship_Type">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Sponsorship Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sponsorship Type</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
